oonimkall/task_test.go

```diff
@@ -431,7 +431,7 @@ func TestIntegrationInterruptNdt7(t *testing.T) {
 		t.Fatal(err)
 	}
 	go func() {
-		<-time.After(7 * time.Second)
+		<-time.After(11 * time.Second)
```

Comment:
This seems to fix an integration test that started failing with GitHub Actions (but not locally)